app/src/main/java/io/github/droidkaigi/confsched2017/model/Speaker.java

```diff
@@ -34,4 +39,20 @@
     @Nullable
     @SerializedName("github_name")
     public String githubName;
+
+    @Nullable
+    public String getAdjustedImageUrl() {
+        if (imageUrl == null) {
+            return null;
+        }
+
+        if (imageUrl.startsWith("http://") || imageUrl.startsWith("https://")) {
```

Comment:
To be honest, I want to use `URLUtil.isNetworkUrl(imageUrl)`, but it depends on Android code.